griptape/drivers/embedding/sagemaker_embedding_driver.py

```diff
@@ -0,0 +1,35 @@
+import json
+import boto3
+from typing import Any
+from attr import define, field, Factory
+from griptape.drivers import BaseEmbeddingDriver
+
+
+@define
+class AmazonSagemakerEmbeddingDriver(BaseEmbeddingDriver):
+    endpoint: str = field(kw_only=True)
```

Comment:
Let's move `model` up to `BaseEmbeddingDriver` and use that instead of declaring a new field.